packages/server/validators/events.js

```diff
@@ -193,8 +194,41 @@ const eventIdValidator = [
     }),
 ];
 
+const attendeeStatusValidator = [
+  // validate status
+  body("status", "invalid status")
+    .trim()
+    .blacklist("<>")
+    .exists({ checkFalsy: true })
+    .withMessage("status cannot be null or empty")
+    .custom(async (status) => {
+      // Validate status
+      const allowedStatusValues = ["NotInterested", "Interested", "Attending"];
+      if (!allowedStatusValues.includes(status)) {
+        throw new Error(
+          "Invalid status value. Allowed values are: NotInterested, Interested, Attending"
+        );
+      }
+    }),
```

Comment:
Don't use a custom validator for this since all you're doing is matching characters which is better suited for regular expressions. Express-validator comes with `.matches()` to validate against a regular expression. See docs [here](https://express-validator.github.io/docs/api/validation-chain/#matches). You can also look at the other validators for examples of regular expression checks that we've already used.